_ont/ont-sercomm-fg1000b-11.md

```diff
@@ -44,14 +46,168 @@ The ONT seems only to display output of the ROM CFE and flash CFE, but don't all
+In the example below you would first creat a /data/up shell script and ensure it has execute rights (ex chmod 755)
+```
+#first we need to add the missing entry
+cmld_client add InternetGatewayDevice.Services.StorageService. 1
+#then inject within the 12 character limit the hostname and a call to our script
+cmld_client set InternetGatewayDevice.Services.StorageService.1.X_SC_NetbiosName='a;/data/up&'
+cmld_client save
+```
+
+## Log configuration
+'syslogd' is configure via Config DB config 'cmld_client get_node InternetGatewayDevice.X_SC_Management.Syslog.' this config is read from the libsl_syslog.so plugin of smd daemon, which generate the file '/tmp/lxxd/logd.conf' and start the daemon with it as parameter.
+
+# GPON ONU status
+
+## Get the operational status of the ONU
+```
+/bin/gponctl getState
+``` 
+
+## Get information of the OLT vendor
+```
+/usr/sbin/umci_ctl stack get olt_type
+```
+or
+```
+/usr/sbin/umci_ctl rg help
+```
+
+## Querying a particular OMCI ME
+```
+/usr/sbin/umci_ctl mib
+```
+
+## Getting/Setting Speed LAN Mode
+
 # GPON/OMCI settings
+
+Part of GPON config is done via the misc configuration loaded as first lib by the smd binary, config can be seens here:
+```
+/usr/bin/cmld_client get_node InternetGatewayDevice.X_SC_MiscCfg.GPON.
+```
+Beware the field 'OmciManageUniMask', 'PretendFwVersion' are initiated in the binary with respective value '01000000', '0'
+
+## Getting/Setting ONU GPON Serial Number
+Default value: 16 hexa on the back of the ONT, starts with '53434F4DA'
+You can test serial and/or ploam combinaison using with below command. Pwd is Hexe only and can be up to 36.
+```
+/bin/gponctl stop
+/bin/gponctl setSnPwd --pwd 00-00-0X-XX-XX-XX-XX-XX-XX-XX --sn YY-YY-YY-YY-YY-YY-YY-YY
+/bin/gponctl start
+
+# You can monitor status by running:
+/bin/gponctl getstate
+```
+To save the serial number you need to re-mount R/W the '/tmp/var_link_dir/ft' and change the "gpon_sn" file (consider backup of the folder before ANY action)
+```
+/bin/mount -o remount,rw /dev/mtdblock5 /tmp/var_link_dir/ft
+echo "XXXXXXXXXXXXX" > /tmp/var_link_dir/ft/gpon_sn
+/bin/mount -o remount,ro /dev/mtdblock5 /tmp/var_link_dir/ft
+/sbin/reboot
+```
+
 ## Getting/Setting ONU GPON PLOAM password
 
-PLOAM can be set directly for Text or Hexa(without 0x) via Web interface if <10 digit otherwise POST call to URL allow > 10 digits for example 20 digit hex can be setup via:
+As of now there was not found a config/command other than the http call below to save the PLOAM password permanently after reboot.
+PLOAM can be set directly for Text or Hexa(without 0x) via Web interface if <10 digit otherwise POST call to URL allow > 10 digits for example 20 digit hex can be setup via (max is 36 digit):
```

Comment:
add space before  (without 0x), and use the ` for the 0x (yes I should have realised this long ago)